src/allure-nunit-tests/allure-results/executor.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Same question

Reply:
I added sample configuration files for this setting so I could see it flow through to the local report. 

Might be a useful reference for anyone looking into producing a fully featured report. 

Happy to amend and remove.

Reply:
I think it's better to remove it

Reply:
This file is usually generated by CI plugins. Same as tests trend, job history, etc.
I'd add this to docs later

Reply:
> This file is usually generated by CI plugins. Same as tests trend, job history, etc.
> I'd add this to docs later

This is good to know. I generate all my reports manually, copy history etc.

Have you integrated into TeamCity / Azure DevOps using this approach?
I'm very interested as I love what Allure can do.